### PR TITLE
Add expression utilities

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio-builder/issues/1529

Here added two utilities:

validateExpression to parse syntax and replace identifier. For parsing used jsep package which is very limited subset of js.

executeExpression to execute the chain of expressions. The code is unoptimized and will not cache results between calls. But still should be fast enough because expressions are indented to be quite simple.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
- [ ] hi @istarkov, I need you to do
  - detailed review (read every line)

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/react-sdk/src/expression.test.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import { expect, test } from "@jest/globals";
+import { executeExpression, validateExpression } from "./expression";
+
+test("allow literals and array expressions", () => {
```
Comment (kof) [inline packages/react-sdk/src/expression.test.ts:4]: can we put that into describe('expression')?

Comment (TrySound) [reply packages/react-sdk/src/expression.test.ts:4]: Jest already groups by module name. This will just add additional nesting without any other benefit.

**Review by kof — APPROVED** [review]

File: packages/react-sdk/src/expression.test.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  expect(
+    executeExpression("exp1", "$ws$dataSource$", variables, expressions)
+  ).toEqual(2);
+});
+
+test("execute expression dependent on variables", () => {
+  const variables = new Map([["var1", 5]]);
+  const expressions = new Map([["exp1", "$ws$dataSource$var1 + 1"]]);
+  expect(
+    executeExpression("exp1", "$ws$dataSource$", variables, expressions)
```
Comment (istarkov) [inline packages/react-sdk/src/expression.test.ts:65]: Any explanation why we need `variablePrefix`
Why not 
```
 const expressions = new Map([["$ws$dataSource$exp1", "1 + 1"]]);
```
Hard to read 
```
  const expressions = new Map([
    ["exp1", "$ws$dataSource$exp0 + 1"],
    ["exp0", "$ws$dataSource$var1 + 2"],
  ]);
  expect(
    executeExpression("exp1", "$ws$dataSource$", variables, expressions)
```

Comment (istarkov) [reply packages/react-sdk/src/expression.test.ts:65]: Somehow I should get from above that $ws$dataSource$exp0 is ($ws$dataSource$var1 + 2)

**Pull request merged**